solo/utils/checkpointer.py

```diff
@@ -135,6 +136,14 @@ def save(self, trainer: pl.Trainer):
                 os.remove(self.last_ckpt)
             self.last_ckpt = ckpt
 
+            current_val = float(trainer.callback_metrics['val_acc1'])
+            best_ckpt = self.path / 'best.ckpt'
```

Comment:
I don't understand this line. What is the / operation for strings?

Reply:
It is not for strings, `/` is overloaded in the `Path` object of `pathlib` apparently. Check line `132`, that's what they do as well.

Reply:
Oh nice!